node_modules/oae-core/manageaccess/js/manageaccess.js

```diff
@@ -29,6 +29,9 @@ define(['jquery', 'oae.core', 'jquery.autosuggest'], function($, oae) {
 
         // Caches the member updates that need to be applied
         var membersUpdates = {};
+        
+        // Caches members pending addition
+        var pendingMembers = {};
```

Comment:
Can you make this an array and call it something like `newUserIds`? Also update the docs on `membersUpdates` to better indicate that the values of the object will be the role updates to apply. That way the purpose of the two objects are better distinguished